From 6f8436709430fd5ebaebd90dd55885636030940d Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Mon, 19 Nov 2018 17:43:24 -0800
Subject: [PATCH] three-grid: don't leak values from hashtable

We are responsible for freeing the GList chain, so ensure we do that at
the end of size-allocate.
---
 src/widgets/dzl-three-grid.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/widgets/dzl-three-grid.c b/src/widgets/dzl-three-grid.c
index 0de5484..d00d0b1 100644
--- a/src/widgets/dzl-three-grid.c
+++ b/src/widgets/dzl-three-grid.c
@@ -569,6 +569,8 @@ dzl_three_grid_size_allocate (GtkWidget     *widget,
       area.y += child_alloc.height + priv->row_spacing;
       area.height -= child_alloc.height + priv->row_spacing;
     }
+
+  g_list_free (values);
 }
 
 static void
-- 
2.17.0

